tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2022.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build e2e
+
+package e2e
+
+import (
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+// It's quite possible these integration tests fail due to maven
+// central blocking requests through rate limiting mechanisms
+
+func runCli(t *testing.T, arg ...string) (string, error) {
+	t.Helper()
+	cmd := "../../validate"
+	c := exec.Command(cmd, arg...)
+	out, err := c.CombinedOutput()
+	return string(out), err
+}
+
+func TestGoodArtifact(t *testing.T) {
+	out, err := runCli(t, "-jar-file", "../resources/jars/maven-wrapper.jar")
```

Comment:
Putting this stuff in a `testdata` directory in the same directory as the tests that use it is more idiomatic.